src/Truncate.js

```diff
@@ -83,7 +83,7 @@ export default class Truncate extends Component {
     // Shim innerText to consistently break lines at <br/> but not at \n
     innerText(node) {
         const div = document.createElement('div');
-        const contentKey = 'innerText' in document.createElement('div') ? 'innerText' : 'textContent';
+        const contentKey = 'innerText' in HTMLElement.prototype ? 'innerText' : 'textContent';
```

Comment:
Hey @UgRoss, this seems like it should have been `'innerText' in window.HTMLElement.prototype` to fix the "ReferenceError: HTMLElement is not defined" errors in the tests